packages/core/echo/echo-pipeline/src/space/space.ts

```diff
@@ -245,8 +246,8 @@ export class Space {
   async initializeDataPipeline() {
     log('initializeDataPipeline');
     assert(this._isOpen, 'Space must be open to initialize data pipeline.');
-    await this._dataPipeline.open();
     assert(this._dataPipelineCredentialConsumer);
     await this._dataPipelineCredentialConsumer.open();
```

Comment:
I'd move this to open() that happens earlier, this way the consumer can start processing epoch messages as they come in